src/CoreWCF.MSMQ/src/CoreWCF/Channels/MsmqBindingBase.cs

```diff
@@ -0,0 +1,116 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.ComponentModel;
+
+namespace CoreWCF.Channels
+{ 
+    public abstract class MsmqBindingBase : Binding
+    {
+        internal MsmqBindingElementBase _transport;
+
+        protected MsmqBindingBase()
+        { }
+
+        // [DefaultValue(typeof(System.TimeSpan), MsmqDefaults.ValidityDurationString)]
+        //public TimeSpan ValidityDuration
```

Comment:
Can you add a comment that this will need to be enabled when we add System.Transactions support?